etc/kayobe/ansible/ubuntu-upgrade.yml

```diff
@@ -63,10 +65,64 @@
           deb {{ stackhpc_repo_ubuntu_noble_security_url }} noble-security main restricted universe multiverse
       become: true
 
-    - name: Do release upgrade
-      ansible.builtin.command: do-release-upgrade -f DistUpgradeViewNonInteractive
+    - name: Enusre /tmp is mounted with exec
+      ansible.builtin.command: sudo mount -o remount,exec /tmp
       become: true
```

Comment:
There is a [mount module](https://docs.ansible.com/ansible/latest/collections/ansible/posix/mount_module.html) which would work well here